### PR TITLE
[TACHYON-1109] Disallow trailing period in @return tag

### PR DESCRIPTION
**Pull request by yupeng9**
https://tachyon.atlassian.net/browse/TACHYON-1109

### CODE REVIEW

**Comment by yupeng9** [comment]
Jenkins, test this please

File: build/checkstyle/tachyon_checks.xml [hunk 1]
```diff
@@ -27,10 +27,15 @@
   <property name="severity" value="error"/>
 
   <!-- Checks for whitespace                               -->
-  <!-- See http://checkstyle.sf.net/config_whitespace.html -->
+  <!-- See http://checkstyle.sf.net/config_whitespace.html -->
```
Comment (gpang) [inline build/checkstyle/tachyon_checks.xml]: extra spaces at the end of the line?

**Comment by calvinjia** [comment]
What's the proposed style for periods/capitalization?

**Comment by yupeng9** [comment]
I think for tags return and param, the sentence should not begin with capitalized, and therefore should not have period at the end.

**Comment by apc999** [comment]
@yupeng9 , there are unclean whitespaces in `tachyon_checks.xml`

**Comment by yupeng9** [comment]
Fixed. @apc999 ,  PTAL.

File: clients/unshaded/src/main/java/tachyon/hadoop/AbstractTFS.java [hunk 2]
```diff
@@ -317,17 +317,18 @@ public FileStatus getFileStatus(Path path) throws IOException {
    * to make loading new FileSystems simpler. This doesn't exist in Hadoop 1.x, so cannot put
    * @Override.
    *
-   * @return schema hadoop should map to.
+   * @return schema hadoop should map to
    *
    * @see org.apache.hadoop.fs.FileSystem#createFileSystem(java.net.URI,
    *      org.apache.hadoop.conf.Configuration)
    */
+  @Override
```
Comment (apc999) [inline clients/unshaded/src/main/java/tachyon/hadoop/AbstractTFS.java]: @yupeng9 could you remove this `@Override` for now.

**Comment by apc999** [comment]
@yupeng9 , could you remove override for this PR?

**Comment by yupeng9** [comment]
@apc999  resolved. PTAL

**Comment by yupeng9** [comment]
Jenkins, test this please

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**